docs: Suggest using alias for minikube kubectl -- (#11997)
kubernetes/minikube: site/content/en/docs/start/_index.md

kubectl get po -A
```

Alternatively, minikube can download the appropriate version of kubectl and you should be able to use it like this:

```shell
minikube kubectl -- get po -A
```

> If you've downloaded kubectl through minikube, you can make your life easier by adding the following to your shell config:

Reviewer: ```suggestion
You can also make your life easier by adding the following to your shell config:
```
